docs/docs/react-without-jsx.md

```diff
@@ -33,16 +35,30 @@ class Hello extends React.Component {
 }
 
 ReactDOM.render(
-  React.createElement(Hello, {toWhat: 'World'}, null),
+  React.createElement(Hello, { toWhat: 'World' }, null),
   document.getElementById('root')
 );
 ```
 
+`React.createElement()` takes three arguments arguments: a component, a `props` object, and any number of children as the final arguments.
+
+The component argument can take a number of forms:
+
+1. A string - `React.createElement("div", null, null)`
+2. A sub-class of `React.Component` - `React.createElement(Hello, null, null)`
+3. A plain stateless-functional componet - `React.createElement(HelloFunctionalComponent, null, null)`
```

Comment:
Let's just unify 2 and 3 under "A React component". Whether it is a class or function doesn't matter to the user.

Reply:
Let's also remove the unnecessary arguments? They look weird here. Or you can make them `, props, children`.